### DISCUSSION TITLE
Framework Integration Status

### DISCUSSION THREAD

**Discussion by ryanbethel** [beginner-corp/begin-community#98] [Frameworks]
There are examples ([here](github.com/begin-examples)) for integrating many popular frontend frameworks with [Begin.com](begin.com). The following is a partial list with their status. Let us know if there is one you would like to see that is not on the list. 

- Svelte: **Working**
  - [example](https://github.com/begin-examples/node-svelte)
- Svelte-Kit: **Working**
  -  [adapter](https://github.com/architect/sveltekit-adapter)
- NextJS: **Working**
  -  [ssr example](https://github.com/begin-examples/next-ssr)
  -  [static example](https://github.com/begin-examples/nextjs-blog-example)
- Astro: **Working**
  - [example](https://github.com/begin-examples/astro-starter)
- Redwood: **In Progress**
- Blitz: **In Exploration**

**Comment by tbeseda** [comment]
Have you seen [Imba](https://imba.io/) @ryanbethel ?
It's a framework and it's own language for both front and back end. Definitely a build step in there.  
From the site:

> Compiles to Javascript
Works with Node + npm
Smart, Minimal Syntax
Built-in DOM Tags & Styles
Amazing Performance

Kinda reminds me of the Coffeescript days. For better or worse 😬  
Anyway, I was curious how it might work with Arc

**Reply by ryanbethel** [reply]
Cool. I haven't played with that yet. Something new to explore.

> Have you seen [Imba](https://imba.io/) @ryanbethel ?
> It's a framework and it's own language for both front and back end. Definitely a build step in there.
> From the site:
> 
> > Compiles to Javascript
> > Works with Node + npm
> > Smart, Minimal Syntax
> > Built-in DOM Tags & Styles
> > Amazing Performance
> 
> Kinda reminds me of the Coffeescript days. For better or worse 😬
> Anyway, I was curious how it might work with Arc
